### DISCUSSION TITLE
What achievements does GitHub have?

### DISCUSSION THREAD

**Discussion by rossjrw** [community/community#21654] [New to GitHub]
<p>The Arctic Code Vault Contributor badge has been on my profile (and many others) for a while, but I’ve just now noticed it’s been upgraded to an achievement with its own icon:</p>
<p><img src="https://user-images.githubusercontent.com/29130152/181066026-cf49d9a6-5226-47df-b89d-2691185c4f65.png" alt="image" data-base62-sha1="duyFoqLCzyd18sq1nT3JlxsxYj5" width="286" height="146"></p>
<p>What other achievements are there?</p>
<p>Also, is there a way to switch this display back to the more professional-looking text-only view? I wouldn’t want potential employers looking at my profile and thinking all I do on GitHub is game the system for fake points.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>We have a list of the badges here:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#displaying-badges-on-your-profile" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#displaying-badges-on-your-profile" target="_blank" rel="noopener">Personalizing your profile - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#displaying-badges-on-your-profile" target="_blank" rel="noopener">//docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I can’t imagine we’re going to have multiple display options for the badges, but if you’d prefer to not have them on your profile, <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#disabling-badges-on-your-profile">you can turn them off.</a></p>

**Comment by Nirzak** [comment]
<p>Switch to the text view is not possible but yes you can disable it from the settings</p>

**Comment by timothyclarke** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a><br>
Everyone that I know thinks of github with a profession mindset. My $0.02 is with <a class="mention" href="/rossjrw">@rossjrw</a> turning this into an achievement gamifies (and so diminished) the status.</p>
